add draft brainrender-napari tutorial (#55)
brainglobe/website: docs/source/tutorials/visualise-atlas-napari.md

## Visualise an atlas in napari

In this tutorial, you will use `brainrender-napari` to visualise parts of the [`mpin_zfish_1um` zebrafish brain atlas](https://doi.org/10.1016/j.neuron.2019.04.034) in the 3D viewer napari through a series of simple clicks. You will need `napari` installed on your computer - please follow [`napari`'s installation instructions to do so](https://napari.org/stable/tutorials/fundamentals/installation.html).

1. Open `napari`
1. Install `brainrender-napari` by selecting `Plugins > Install/Uninstall plugins` and searching for `brainrender-napari` in the searchbox. Then click on the "Install" button.
1. Open the `Brainrender` widget by selecting `Plugins > Brainrender (brainrender-napari)`. 
    * A widget should appear on the right hand side.

> 1. (You can skip this step if you've already used the `mpin_zfish_atlas` in Brainglobe before) In the `Brainrender` widget's "Atlas table view", double-click the 6th row, which contains the `mpin_zfish_1um` atlas. Answer "Yes" to the appearing dialog (this may take a while). 
>     * You have now downloaded the atlas.
> 1. In the `Brainrender` widget's "Atlas table view", double-click the 6th row, which contains the `mpin_zfish_1um` atlas (again). 
>     * You have now added the annotations image and the reference image to napari.
>     * A "Structures" widget appears below the existing widget.

Reviewer: I find this slightly hard to parse, it says to skip this step (double click), but then the next step is to double click. I find having a shorter list, then a separate box explaining the download easier to parse (not sure if you agree). I also removed reference to the 6th row, as that is likely to change in the future. 

```suggestion
1. In the `Brainrender` widget's "Atlas table view", double-click the row which contains the `mpin_zfish_1um` atlas
    * You have now added the annotations image and the reference image to napari.
    * A "Structures" widget appears below the existing widget.

:::{note}
If you haven't downloaded the atlas before, the plugin will prompt you to download it (click yes). It may take a long time
(depending on your internet speed) so please be patient. Once downloaded, double clicks the atlas to view.
:::
```

> 1. Toggle the visualisation to a 3D view by pressing `Ctrl+Y` (`Cmd+Y` on Mac) on your keyboard.

Reviewer: Could you add details about the button to click, and then say "alternatively press ....". I think button clicking is more intuitive for many users.

Author: I've switched to an entirely mouse-clickey way of doing this, with a picture.

> 1. Navigate the structures tree in the "Structures" widget by opening "forebrain". Double-click on "telencephalon".

Reviewer: I don't think "Structures widget" is clear for many users. It doesn't say Structures anywhere, and they probably don't know what a widget is.

Author: It does say "Structures" now that 
I've basically adapted terminology from `bg-atlasapi` but happy change the user-facing part to something better?
What would a better term be? "Regions"? "Subvolumes"? "Meshes"?

Reviewer: Maybe `3D brain region meshes` to be super clear?

Author: Cool - to be addressed by https://github.com/brainglobe/brainrender-napari/pull/82

    * You have now added a mesh layer of a brain structure to napari.

> 1. Back in the "Atlas table view", right-click on the 6th row. In the appearing context menu, select "GAD1b"

Reviewer: ```suggestion
1. Back in the "Atlas table view", right-click on the `mpin_zfish_1um` row. In the menu that appears, select `GAD1b`
```

    * You have now added an additional reference image to napari

> You have now added all possible types of brainglobe atlas parts (annotations image, reference image, structure mesh, additional reference) to napari - well done! You can now add any of the other atlases listed in the "Atlas table view" if you like (note that not all atlases have additional references!). Hover over any of the elements in the `Brainrender` widget to get additional hints about how to use them!

Reviewer: ```suggestion
You have now added all possible types of BrainGlobe atlas parts (annotations image, reference image, structure mesh, additional reference) to napari - well done! You can now add any of the other atlases listed in the "Atlas table view" if you like (note that not all atlases have additional references!). Hover over any of the elements in the `brainrender` widget to get additional hints about how to use them!
```

## Visualise an atlas in napari

> BrainGlobe atlases are made of different kinds of parts. These include the annotations image, the (default) reference image, 3D brain region meshes and, optionally, additional reference images. In this tutorial, you will use `brainrender-napari` to visualise each of these parts in the 3D viewer napari through a series of simple clicks. The [`mpin_zfish_1um` zebrafish brain atlas](https://doi.org/10.1016/j.neuron.2019.04.034) will serve as an example. 

Reviewer: ```suggestion
BrainGlobe atlases are made of different components. These include the annotations image, the (default) reference image, 3D brain region meshes and, optionally, additional reference images. In this tutorial, you will use `brainrender-napari` to visualise each of these components in the 3D viewer napari through a series of simple clicks. The [`mpin_zfish_1um` zebrafish brain atlas](https://doi.org/10.1016/j.neuron.2019.04.034) will serve as an example. 
```

:::{note}
You will need `napari` installed on your computer - please follow [`napari`'s installation instructions to do so](https://napari.org/stable/tutorials/fundamentals/installation.html).
:::

> 1. Open `napari`

Reviewer: ```suggestion
1. Open `napari`.
```

> 2. Install `brainrender-napari` by selecting `Plugins > Install/Uninstall plugins` and searching for `brainrender-napari` in the searchbox. Then click on the "Install" button.

Reviewer: ```suggestion
2. Install `brainrender-napari` by selecting `Plugins > Install/Uninstall plugins` and searching for `brainrender-napari` in the searchbox. Then click on the `Install` button.
```

3. Open the `brainrender` widget by selecting `Plugins > Brainrender (brainrender-napari)` in the napari menu bar near the top left of the window. 
![brainrender widget](./images/brainrender-napari/plugin-menu-brainrender-napari.png)
**The brainrender widget appears on the right hand side of the window.**

> 4. In the `Brainrender` widget's "Atlas table view" section, double-click the row which contains the `mpin_zfish_1um` atlas

Reviewer: ```suggestion
4. In the `brainrender` widget's `Atlas table view` section, double-click the row which contains the `mpin_zfish_1um` atlas.
```

![brainrender widget with added annotations](./images/brainrender-napari/added-brainrender-napari.png)

> **You have now added the annotations image and the default reference image to napari: They appear as layers in the napari layers list on the lower left of the window. A "3D Brain region meshes" section appears below the "Atlas table view" section.**

Reviewer: ```suggestion
**You have now added the annotations image and the default reference image to napari: They appear as layers in the napari layers list on the lower left of the window. A `3D Brain region meshes` section appears below the `Atlas table view` section.**
```

:::{note}
If you haven't downloaded the atlas before, the plugin will prompt you to download it (click yes). It may take a long time
(depending on your internet speed) so please be patient. Once downloaded, double click again to view the atlas.
:::

5. Toggle the napari display from 2D to 3D by pressing the button with the square icon on the lower left of the window.
![brainrender widget with 3d display](./images/brainrender-napari/toggle-ndisplay-brainrender-napari.png)
**The annotations image should now be displayed in 3D.**

> 6. Navigate the brain region tree in the "3D Brain region meshes" section by opening "forebrain". Double-click on "telencephalon".

Reviewer: ```suggestion
6. Navigate the brain region tree in the `3D Brain region meshes` section by opening "forebrain". Double-click on `telencephalon`.
```

![brainrender widget with region mesh](./images/brainrender-napari/add-region-brainrender-napari.png)

>  **You have now added a 3D brain region mesh layer, which appears as a mesh in the viewer and as a new layer in the layers list**

Reviewer: ```suggestion
 **You have now added a 3D brain region mesh layer, which appears as a mesh in the viewer and as a new layer in the layers list.**
```

> 7. Back in the "Atlas table view" section, right-click on the `mpin_zfish_1um` row. In the menu that appears, select `GAD1b`

Reviewer: ```suggestion
7. Back in the "Atlas table view" section, right-click on the `mpin_zfish_1um` row. In the menu that appears, select `GAD1b`.
```

![brainrender widget with additional reference](./images/brainrender-napari/additional-reference-brainrender-napari.png)

> **You have now added an additional reference image, which appears as a grey scale image in the viewer and as a new layer in the layers list**

Reviewer: ```suggestion
**You have now added an additional reference image, which appears as a grey scale image in the viewer and as a new layer in the layers list.**
```

> You have now added all possible kinds of BrainGlobe atlas parts (annotations image, reference image, 3D brain region mesh, additional reference) to napari - well done! You can now add any of the other atlases listed in the "Atlas table view" if you like (note that not all atlases have additional references!). Hover over any of the elements in the `brainrender` widget to get additional hints about how to use them!

Reviewer: ```suggestion
You have now added all possible kinds of BrainGlobe atlas components (annotations image, reference image, 3D brain region mesh, additional reference) to napari - well done! You can now add any of the other atlases listed in the `Atlas table view` if you like (note that not all atlases have additional references!). 

:::{note}
Hover over any of the elements in the `brainrender` widget to get additional hints about how to use them!
:::
```
